data/speakers.ts

```diff
@@ -6042,6 +6042,26 @@ const speakers: Speaker[] = [
       city: 'New York',
     },
   },
+  {
+    objectID: 274,
```

Comment:
@dotenorio I'm changing ID to smt more human-readable, I'm adding detail page to the app, so your page is going to be like `confcitizens.com/dotenorio`. Or you can choose whatever you want username :)

```suggestion
    objectID: 'dotenorio',
```